integration/run_test.go

```diff
@@ -185,3 +191,216 @@ func testWebServiceContainer(shell *shell.Shell, containerName string) {
 		"curl", "--retry", "5", "--retry-connrefused", "--retry-max-time", "30", "http://127.0.0.1",
 	)
 }
+
+type retryConfig struct {
+	MaxRetries         int
+	MinWaitMsec        int64
+	MaxWaitMsec        int64
+	NetworkDisableMsec int64
+	ExpectedSuccess    bool
+}
```

Comment:
nit: they can be unexported fields

Reply:
Good catch. Eventually this should get refactored to contain a BlobConfig, but that didn't seem necessary yet.